eventuals/collect.h

```diff
@@ -4,6 +4,7 @@
 
 #include "eventuals/loop.h"
 #include "eventuals/type-traits.h"
+#include "google/protobuf/repeated_field.h"
```

Comment:
I'm confused: how does this `#include` succeed? I don't see any dependencies on something like `com_google_protobuf`.

Ah, I'm guessing we're pulling this in from an indirect dependency on `stout`: https://github.com/3rdparty/stout-flags/blob/fdff8dd353991afc193f77506f04da505237a796/bazel/repos.bzl

Can you please add a direct dependency on a relevant proto target to this file's build rule so the source of the dependency is more clear?

Reply:
Done, please review the files again and resolve the conversation if everything is ok!

Reply:
Looks excellent, thanks!